azkustoingest/file_options.go

```diff
@@ -229,16 +229,18 @@ func InferFormatFromFileName(fName string) DataFormat {
 // IngestionMapping provides runtime mapping of the data being imported to the fields in the table.
 // "ref" will be JSON encoded, so it can be any type that can be JSON marshalled. If you pass a string
 // or []byte, it will be interpreted as already being JSON encoded.
-// mappingKind can only be: CSV, JSON, AVRO, Parquet or ORC.
+// mappingKind can only be: CSV, JSON, AVRO, Parquet or ORC, but it will be automatically converted from other types.
 // The mappingKind parameter will also automatically set the FileFormat option.
```

Comment:
First line is just confusing
```suggestion
// The format parameter will automatically set the FileOption.Format option.
```